src/style.css

```diff
@@ -1,3 +1,86 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  padding-left: 50px;
+  height: 60px;
```

Comment:
Let's set the height to the link itself, not the header

![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/a95b7154-9e8d-46fe-9e9c-411f2d44892a)